app/styles/_shame.scss

```diff
@@ -131,3 +63,47 @@ $au-non-editable-background    : $au-gray-100 !default;
     background: $au-input-disabled-color;
   }
 }
+
+/******************************************************************************
+  Pagination on the mock-login (possibly among others)
+******************************************************************************/
+
+.data-table-pagination {
```

Comment:
I saw that, but it doesn't provide the same functionality. I would miss the select for the pages. When logging in for the "Gemeente Mechelen" I know I have to go to page 3 in one action. This can't be done with the new component.

I will change it to the component you suggest, because that is even less styling and it is a supported component by ember-appuniversum.

Reply:
Yea agreed, but since it's only for the mock login I don't think it matters that much. I'm not sure why the page selector isn't present in the Appuniversum equivalent.